app/models/container_project.rb

```diff
@@ -78,4 +78,8 @@ def disconnect_inv
     self.deleted_on = Time.now.utc
     save
   end
+
+  def archived?
+    ems_id.nil?
+  end
```

Comment:
@zeari Not relevant for this PR but do we use this to decide what to display in projects explorer view?